### ISSUE TITLE
About Dataset Preparation issue

### ISSUE THREAD

**Issue by lixiangyu-1008** [swathikirans/ego-rnn#4]
Hi @swathikirans  thanks for your great work!

I follow the download link https://drive.google.com/file/d/1_y8Y3PnCXsngmZVMqZbg-AfJyIdOeQ2_/view?usp=sharing

But I cannot find the **--trainDatasetDir ./dataset/gtea_61/split2/train** in the zip file. It only have the "flow_x", "flow_y" and "frames".

**Comment by swathikirans** [comment]
I shared the frames of all the videos in the dataset. You can create the train/val splits by yourself.  For example, the train data of split 2 consists of samples from S1, S3 and S4 while the val split consists of samples from S2.

**Comment by lixiangyu-1008** [comment]
Thanks for replying.
So how I can set the correct the train data of split 2 as you said in your paper?
Is it the train data of split 2 only sample from S2?

**Comment by swathikirans** [comment]
For split 2, copy(or make symbolic links of) dirs S1, S3 and S4 to the train dir and S2 to the test dir

**Comment by lixiangyu-1008** [comment]
Thanks, I  will have a try!

**Comment by lixiangyu-1008** [comment]
Hi @swathikirans 
What's the different about the GTEA 61* and GTEA 61, it means that how to the fixed split?

**Comment by swathikirans** [comment]
GTEA61* is split 2 and GTEA61 in the table is the average across all four splits

**Comment by lixiangyu-1008** [comment]
Is that for GTEA61, we should setting the train/test split as follows:
For split1 :   training : S2,S3,S4.    testing:S1
For split2 :   training : S1,S3,S4.    testing:S2
For split3 :   training : S1,S2,S4.    testing:S3
For split4 :   training : S1,S2,S3.    testing:S4
Then average the four split accuracy ?

**Comment by swathikirans** [comment]
Yes, you are right.

**Comment by lixiangyu-1008** [comment]
Thanks! :) 
Have you plan to test your model on **EPIC-KITCHENS Dataset** ?

**Comment by swathikirans** [comment]
No, I don't think I will try this model on EPIC-KITCHENS dataset.

**Comment by lixiangyu-1008** [comment]
Ok, Thanks, I will follow your LSTA on EPIC-KITCHENS dataset.

**Comment by frostinassiky** [comment]
@lixiangyu-1008 Any follow-up?

**Comment by chenwenhe11** [comment]
I have the same problem, how did you solve it？

**Comment by chenwenhe11** [comment]
> Hi @swathikirans thanks for your great work!
> 
> I follow the download link https://drive.google.com/file/d/1_y8Y3PnCXsngmZVMqZbg-AfJyIdOeQ2_/view?usp=sharing
> 
> But I cannot find the **--trainDatasetDir ./dataset/gtea_61/split2/train** in the zip file. It only have the "flow_x", "flow_y" and "frames".

I have the same problem, how did you solve it

**Comment by chenwenhe11** [comment]
@lixiangyu-1008 I have the same problem, how did you solve it？
